example.dev-config.yml

```diff
@@ -4,6 +4,10 @@ pulp_default_admin_password: password
 pulp_install_plugins:
   # pulp-ansible:
   #   source_dir: "/home/vagrant/devel/pulp_ansible"
+  # galaxy-pulp:
```

Comment:
How'd you come up w/ this name?

Reply:
it is the name of their bindings: https://github.com/ansible/galaxy_ng/blob/master/bindings/build.sh#L11